changelog/allMembers_import.dd

```diff
@@ -0,0 +1,7 @@
+The allMembers trait behaves more correctly with imports
+
+1. `import`s are presented using their fully qualified name.
+Previously only the first identifier of the import was present in the list,
+e.g `std` for `std.stdio`.
+
+2. members of a package, e.g `std.algorithm` or `std.range` can be retrieved.
```

Comment:
Again, please provide an example - I assume that now `pragma (msg, [__traits(allMembers, std.algorithm)]);` would print the FQN of the 6 modules (as IIRC std.algorithm doesn't have any direct members).

Reply:
ok.